app/views/pages/index.html.erb

```diff
@@ -2,6 +2,5 @@
   You are logged in as <%= current_user.email %><br>
   <%= link_to 'Log Out', logout_path, method: :delete %>
 <% else %>
-  <%= link_to 'Log In', new_session_path %><br>
-  <%= link_to 'Register', new_user_path %>
+  <%= link_to 'Log In', sign_in_path %><br>
 <% end %>
```

Comment:
You could take this logic out pretty easily. You could move this to a helper filer and in it's place leave <%=logged_in?%> and  <%= login_or_logout_button%>, which would call upon the appropriate methods.